packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -282,6 +282,7 @@ devServerFeature.setup(
              *  but we will keep on watching on the users application
              *  2. the createCompiler function is not extendable with more configs with the current API
              */
+            console.time('Bundling files')
```

Comment:
Should the `console.log` exist? 
if so, consider being more specific about the task (IMO "files" is quite vague)